plugins/defaultplugins/ifplugin/interface_config.go

```diff
@@ -179,6 +186,20 @@ func (plugin *InterfaceConfigurator) ConfigureVPPInterface(iface *intf.Interface
 		}
 	}
 
+	// configure mtu
+	var mtu uint32
+	if iface.Mtu != 0 {
+		mtu = iface.Mtu
+	} else if plugin.cfgMtu != 0 {
+		mtu = plugin.cfgMtu
+	} else {
+		mtu = defaultMtu
```

Comment:
What about doing cfgMtu vs. const defaultMtu rigth after loading the configuration in Init() rather the per each iface?